openhtf/util/argv.py

```diff
@@ -41,3 +41,35 @@ def __call__(self, parser, namespace, values, option_string=None):
     else:
       module = __import__(self._tgt_mod)
     setattr(module, self._tgt_attr, values)
+
+
+class _StoreValueInModule(argparse.Action):
```

Comment:
merge with storeinmodule?

Reply:
Merged as much as possible, following argparse's _StoreConstAction, _StoreTrueAction, and _StoreFalseAction.